src/main/java/org/matsim/run/prepare/PrepareDrtStops.java

```diff
@@ -0,0 +1,27 @@
+package org.matsim.run.prepare;
+
+import org.matsim.api.core.v01.Scenario;
+import org.matsim.api.core.v01.network.Network;
+import org.matsim.core.config.Config;
+import org.matsim.core.config.ConfigUtils;
+import org.matsim.core.scenario.ScenarioUtils;
+
+import java.io.IOException;
+
+public class PrepareDrtStops {
```

Comment:
This class should be app command with options, so it can be used in the preparation pipeline.